queries/sql/indents.scm

```diff
@@ -7,8 +7,7 @@
  (insert)
 ] @indent.begin
 
-
-(compound_statement
+(block
```

Comment:
Hmmm, which one is more narrowed down. Is it `compound_statement` or `block`

as in, is the s-expr tree `(compound_statement (block (keyword_begin)))` or `(block (compound_statement (keyword_begin)))`

Reply:
Ahhh, I see from the commit logs of tree-sitter-sql now, `compound_statement` is now renamed and refactored into `block`, is it correct? @dmfay

Reply:
I think this was just a rename in the grammar?

Reply:
that's correct, `block` is a more common name for the expression formerly called `compound_statement`. Only the name has changed, nothing's more specific or more general.